packages/components/src/AppLoader/constant.js

```diff
@@ -44,42 +26,73 @@ body {
 	margin: auto;
 }
 
-.tc-app-loader:before,
-.tc-app-loader:after {
-	animation: app-loader-spin 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
-	box-sizing: border-box;
-	content: '';
-	display: block;
+.tc-app-loader-animate {
+	width: 205px;
+	height: 205px;
 	position: absolute;
-	width: 165px;
-	height: 165px;
-	border: 4.5px solid #ababab;
-	border-radius: 50%;
-	margin: -7px 0 0 -7px;
-	border-color: #ababab transparent transparent transparent;
+	margin: -27px 0 0 -27px;
+	animation: rotate 2s linear infinite;
 }
 
-.tc-app-loader:before {
-	animation-delay: -0.3s;
+.tc-app-loader-path {
+	stroke-dasharray: 12.5664; 
+	stroke-dashoffset: 0;
+	stroke: #ababab;
```

Comment:
This is `$silver-chalice`

Reply:
we cannot use scss variables as this is a plain css